docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,18 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms` aggs>> and 
+model plot is not enabled for the {anomaly-job}, neither the **Single Metric 
+Viewer** nor the **Anomaly Explorer** can plot and display an anomaly 
+chart for the job. In these cases, the charts are blank.
```

Comment:
My apologies for another change, for 7.11, in these cases, the charts will not be visible and there will be call outs explaining why.

Reply:
Modified the text accordingly via 5148e84.